examples/iota_lattice/analysis_iotalattice.py

```diff
@@ -78,15 +78,14 @@ def get_moments(beam):
 )
 
 atol = 0.0  # a big number
-rtol = 1.5 * num_particles**-0.5  # from random sampling of a smooth distribution
+rtol = 1.8 * num_particles**-0.5  # from random sampling of a smooth distribution
```

Comment:
Bumped the tolerance slightly after removing `sigt`. Now it passes.

Is the range still reasonable to you?

Reply:
Yes.